sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs

```diff
@@ -344,6 +365,34 @@ private void InitializePlaywrightServiceEnvironmentVariables(string? os = null,
         }
         SetReportingUrlAndWorkspaceId();
     }
+    private void WarnIfAccessTokenCloseToExpiry()
+    {
+        string accessToken =  GetAuthToken()!;
+        if (string.IsNullOrEmpty(accessToken))
+            throw new Exception(Constants.s_no_auth_error);
```

Comment:
No need to throw an exception, that will be done before. If the code flow is here, that means access token exists.